app.py

```diff
@@ -54,6 +67,21 @@ def load_post_ymls_by_tag(tag) -> dict:
 
 @dataclass
 class Post:
+
+    """Class representing a single post
+
+    Attributes:
```

Comment:
Commends would probably look better if the first letter of each description is capitalized, like usual text.